charts/v2.0.0-alpha.1/azuredisk-csi-driver/templates/csi-azuredisk-controller-cleanup.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: batch/v1
```

Comment:
Can we resuse this template for deleting resources viahelm uninstall for the sanity and integration tests?

Reply:
I was hoping to do so but because of different api-server for the created minikube / kind cluster, it is not so straightforward. I am looking for ways to uniformize clean-up methods but it will have to stay this way for now unfortunately.